### PR TITLE
[test] Add specific tag for MCH tests

### PR DESCRIPTION
**Pull request by vkarak**
Also:
  - Remove `kesch:cn` from CDO/NCO tests

### CODE REVIEW

**Comment by vkarak** [comment]
@jenkins-cscs retry all

**Comment by vkarak** [comment]
@ajocksch @omlins @teojgo Can sb review this? The idea is to easily select MCH tests.

File: cscs-checks/tools/io/cdo.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def __init__(self):
         self.sourcesdir = os.path.join(self.current_system.resourcesdir,
                                        'CDO-NCO')
         self.valid_systems = ['daint:gpu', 'daint:mc', 'dom:gpu', 'dom:mc',
-                              'kesch:pn', 'kesch:cn']
+                              'kesch:pn']
```
[the remark this answers is not in the document]
Comment (omlins) [reply cscs-checks/tools/io/cdo.py]: Yes, I would remove 'kesch:cn' either from both CDO and NCO or keep it for both. Why do you want to remove it actually? Does it take a long time to run? Are you sure that MCH doesn't want to use CDO and NCO on the compute nodes (the tests verify the correct installation which is not always in sync between compute nodes and login nodes)?

[the remark this answers is not in the document]
Comment (vkarak) [reply cscs-checks/tools/io/cdo.py]: I agree for the symmetry. It's an omission. Regarding the run time, MCH is strict, so we should keep the tests at what exactly is required. I think that CDO/NCO are postprocessing tools, aren't they? Does MCH run them on the `cn` partition as well? If you know that, then I'm ok with reverting the changes.

[the remark this answers is not in the document]
Comment (omlins) [reply cscs-checks/tools/io/cdo.py]: No, I don't know what are MCH's requirements.

**Comment by vkarak** [comment]
@jenkins-cscs retry kesch

**Pull request merged**